core/build/src/lib.rs

```diff
@@ -17,8 +30,11 @@ static DEFAULT_SGX_SDK_PATH: &str = "/opt/intel/sgxsdk";
 ///
 /// # Arguments
 /// * `name` - The name of the type to determine the bindgen name of.
-pub fn sgx_normalize_item_name(name: &str) -> Option<String> {
-    if name.starts_with("_sgx") || name.starts_with("_tee") {
+pub fn normalize_item_name(name: &str) -> Option<String> {
+    if STRIP_UNDERSCORE_PREFIX
+        .iter()
+        .any(|prefix| name.starts_with(prefix))
+    {
         Some(name[1..].to_owned())
```

Comment:
Suggest using [strip_prefix](https://doc.rust-lang.org/std/primitive.str.html#method.strip_prefix) here for clarity of intent (not critical).

Reply:
Done, [afca2be](https://github.com/mobilecoinfoundation/sgx/pull/53/commits/afca2be60616178408231d1fafcd789cfdb633d7)